### DISCUSSION TITLE
Configuration to omit generation of claims data

### DISCUSSION THREAD

**Discussion by garyisaac** [synthetichealth/synthea#1397] [Q&A] [answered]
We are having issues loading Synthea test patients on a FHIR server that supports US Core 3.1.1. I pulled an older version of Synthea that supports US Core 3.1.1. I believe it is a April 2023 branch that was recommended in another post in Discussions. Is there a configuration in the properties files that restrict claims data from being exported? The server we are using does not support claims because they say that resource is not in 3.1.1.

Thank you!

**Comment by dehall** [comment, marked as answer]
Yes, there is a way to filter out certain resource types. The config setting is `exporter.fhir.excluded_resources` which you can either set in `src/main/resources/synthea.properties` or on the command line, ex `./run_synthea --exporter.fhir.excluded_resources=Claim`.  This setting takes a comma-separated list of resource types if you want to exclude more than one, for example `exporter.fhir.excluded_resources=Claim,ExplanationOfBenefit,Provenance`

This feature was added in November 2022 so it should be available in the version you have (https://github.com/synthetichealth/synthea/pull/1178)

**Comment by garyisaac** [comment]
Great! Thank you Dylan!
